src/Makefile

```diff
@@ -64,6 +64,9 @@ FLAGS += -DUSE_POLLY
 ifeq ($(USE_POLLY_OPENMP),1)
 FLAGS += -fopenmp
 endif
+ifeq ($(USE_POLLY_ACC),1)
+LLVMLINK += -lPollyPPCG -Wl,-rpath=$(shell $(LLVM_CONFIG_HOST) --libdir),--no-as-needed -lGPURuntime -Wl,--as-needed
```

Comment:
When Polly-ACC is used with clang, calls to runtime functions (in GPURuntime) that facilitate the GPU offload are inserted into the resulting binary. In this [example](http://polly.llvm.org/documentation/gpgpucodegen.html), this library is made available through LD_LIBRARY_PATH.

In the case of libjulia.so, there aren't any calls to functions present in GPURuntime. The --no-as-needed option's therefore needed to prevent it being optimzed away. The location of GPURuntime has to be given in the case of external LLVM builds. This is done using -rpath.